### ISSUE TITLE
There is no Metrics under Features in cluster settings page

### ISSUE THREAD

**Issue by zzswang** [lensapp/lens#2284]
**Describe the bug**
There is no Metrics under Features in cluster settings page

**To Reproduce**
1. Fresh install lens 4.1.4
2. Add a cluster just created by k3sup

**Expected behavior**
There should be metrics button in Features setting block.

**Screenshots**

![image](https://user-images.githubusercontent.com/1524745/110127211-51a80e00-7e00-11eb-9053-9740b3e14ebe.png)

**Environment (please complete the following information):**
- Lens Version: 4.1.4
- OS: OSX 10.15.7
- brew install lens

**Lens on Windows connected to the same cluster works fine**

![image](https://user-images.githubusercontent.com/1524745/110127313-74d2bd80-7e00-11eb-9f20-6b68e5af69fe.png)

**Comment by jim-docker** [comment]
Can you provide logs? (https://docs.k8slens.dev/v4.0.6/faq/#where-can-i-find-application-logs)

**Comment by zzswang** [comment]
```
╰─ tail -f lens.log       
info: [STORE]: SAVING /Users/zzs/Library/Application Support/Lens/lens-cluster-store.json
info: [STORE]: SAVING /Users/zzs/Library/Application Support/Lens/lens-cluster-store.json
info: [STORE]: SAVING /Users/zzs/Library/Application Support/Lens/lens-cluster-store.json
info: [CLUSTER]: refresh {"id":"dc7fc770-cb9a-40a5-8f58-85b0d8de9a34","name":"pdairport","initialized":true,"ready":true,"online":true,"accessible":true,"disconnected":false}
info: [CLUSTER]: refresh {"id":"dc7fc770-cb9a-40a5-8f58-85b0d8de9a34","name":"pdairport","initialized":true,"ready":true,"online":true,"accessible":true,"disconnected":false}
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
info: [CLUSTER]: refresh {"id":"dc7fc770-cb9a-40a5-8f58-85b0d8de9a34","name":"pdairport","initialized":true,"ready":true,"online":true,"accessible":true,"disconnected":false}
info: [CLUSTER]: refresh {"id":"dc7fc770-cb9a-40a5-8f58-85b0d8de9a34","name":"pdairport","initialized":true,"ready":true,"online":true,"accessible":true,"disconnected":false}
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
```

**Comment by zzswang** [comment]
```
─ tail -f lens1.log                                                                                                                                                                            
info: [KUBE-AUTH]: out-channel "kube-auth:b62ec589-6b1f-4a50-b9ff-67844d8d0129" {"data":"2021/03/09 19:02:52 http: proxy error: x509: certificate signed by unknown authority\n","error":true,"meta":{"id":"b62ec589-6b1f-4a50-b9ff-67844d8d0129","name":"pdairport","initialized":true,"ready":false,"online":false,"accessible":false,"disconnected":false}}
error: Failed to connect cluster "pdairport": StatusCodeError: 502 - undefined
info: [CLUSTER]: refresh {"id":"b62ec589-6b1f-4a50-b9ff-67844d8d0129","name":"pdairport","initialized":true,"ready":false,"online":false,"accessible":false,"disconnected":false}
info: [KUBE-AUTH]: out-channel "kube-auth:b62ec589-6b1f-4a50-b9ff-67844d8d0129" {"data":"2021/03/09 19:03:22 http: proxy error: x509: certificate signed by unknown authority\n","error":true,"meta":{"id":"b62ec589-6b1f-4a50-b9ff-67844d8d0129","name":"pdairport","initialized":true,"ready":false,"online":false,"accessible":false,"disconnected":false}}
error: Failed to connect cluster "pdairport": StatusCodeError: 502 - undefined
info: [CLUSTER]: refresh {"id":"b62ec589-6b1f-4a50-b9ff-67844d8d0129","name":"pdairport","initialized":true,"ready":false,"online":false,"accessible":false,"disconnected":false}
info: [KUBE-AUTH]: out-channel "kube-auth:b62ec589-6b1f-4a50-b9ff-67844d8d0129" {"data":"2021/03/09 19:03:52 http: proxy error: x509: certificate signed by unknown authority\n","error":true,"meta":{"id":"b62ec589-6b1f-4a50-b9ff-67844d8d0129","name":"pdairport","initialized":true,"ready":false,"online":false,"accessible":false,"disconnected":false}}
info: [LENS-VIEW]: init dashboard, clusterId=dc7fc770-cb9a-40a5-8f58-85b0d8de9a34
info: [LENS-VIEW]: waiting cluster to be ready, clusterId=dc7fc770-cb9a-40a5-8f58-85b0d8de9a34
info: [LENS-VIEW]: loaded from http://dc7fc770-cb9a-40a5-8f58-85b0d8de9a34.localhost:56175/
```

**Comment by zzswang** [comment]
certificate err is caused by remote cluster, fixed later

**Comment by jim-docker** [comment]
I did reproduce this, and after reloading (Cmd-R) the Metrics Stack appeared. But can't seem to reproduce it again...

**Comment by jakolehm** [comment]
Most likely fixed in #2714
